websites/K/Kick/presence.ts

```diff
@@ -0,0 +1,509 @@
+const presence = new Presence({ clientId: "1112901248421732462" });
+presence.on("UpdateData", () => {
+	const presenceData: PresenceData = {
+			largeImageKey: "app_icon",
+			startTimestamp: Math.floor(Date.now() / 1000),
+		},
+		pathDetailsMap = {
+			//	  "/": { details: "Viewing", state: "Startpage" },
+			"/terms-of-service": {
+				details: "Reading:",
+				state: "Terms of Service",
+				largeImageKey: "",
+			},
+			"/privacy-policy": {
+				details: "Reading:",
+				state: "Privacy Policy",
+				largeImageKey: "",
+			},
+			"/dmca-policy": {
+				details: "Reading:",
+				state: "DMCA Policy",
+				largeImageKey: "",
+			},
+			"/categories/games/fortnite": {
```

Comment:
There seems to be a lot of categories. Preferably find ways to grab the names/images using document queries, rather than hardcoding them.